src/main/java/org/eclipse/yasson/internal/ClassParser.java

```diff
@@ -120,8 +121,17 @@ private void parseMethods(Class<?> clazz, JsonbAnnotatedElement<Class<?>> classE
                 continue;
             }
             final String propertyName = toPropertyMethod(name);
-
-            Property property = classProperties.computeIfAbsent(propertyName, n -> new Property(n, classElement));
+            Property property = classProperties.get(propertyName);
+            if(property == null) {
```

Comment:
I don't see a point getting and checking for null, since putIfAbsent is used later.

Reply:
Yes, but the key used to put the value in the map is different.